src/app/pages/din-oversikt-page/DinOversiktPage.tsx

```diff
@@ -36,26 +45,85 @@ const Oversikt = ({ søknader }: Props) => {
     const pleiepengesoknader = søknader.filter((søknad) => erPleiepenger(søknad));
     const harSøknader = pleiepengesoknader.length > 0;
     const seksFørsteSoknader = pleiepengesoknader.slice(0, 5);
+    const harArbeidsgiver = (søknad: Søknad) => {
+        if ('arbeidsgivere' in søknad.søknad) {
+            return 'organisasjoner' in søknad.søknad.arbeidsgivere
+                ? søknad.søknad.arbeidsgivere.organisasjoner && søknad.søknad.arbeidsgivere.organisasjoner.length > 0
+                : søknad.søknad.arbeidsgivere && søknad.søknad.arbeidsgivere.length > 0;
+        }
+        return false;
+    };
+    const visAlertstripe =
+        harSøknader &&
+        pleiepengesoknader.some(
+            (søknad) =>
+                søknad.søknadstype === Søknadstype.PP_SYKT_BARN &&
+                harArbeidsgiver(søknad) &&
+                mindreTimerEtterInnsendtEnnMaxAntallTimer(søknad.opprettet, 48)
+        );
 
     useLogSidevisning(PageKey.frontpage);
 
     return (
         <InnsynPage
             title={intlHelper(intl, 'page.dinOversikt.title')}
-            topContentRenderer={() => (
-                <PageBanner title={intlHelper(intl, 'page.dinOversikt.title')} illustration={<SvgSykdomIFamilien />} />
-            )}>
+            topContentRenderer={() => <PageBanner title={intlHelper(intl, 'page.dinOversikt.title')}></PageBanner>}>
+            {visAlertstripe && (
+                <Box padBottom="xl">
+                    <Alertstripe type="advarsel">
+                        <FormattedMessage id="page.pleiepengesakSøknad.søknad.alertstripe.title" />
+                        <ul>
+                            <li>
+                                <FormattedMessage id="page.pleiepengesakSøknad.søknad.alertstripe.list.1" />
+                            </li>
+                            <li>
+                                <FormattedMessage id="page.pleiepengesakSøknad.søknad.alertstripe.list.2" />
+                            </li>
+                        </ul>
+                        <FormattedMessage id="page.pleiepengesakSøknad.søknad.alertstripe.info" />
+                    </Alertstripe>
+                </Box>
+            )}
             <MellomlagringDataFetcher />
-            <div className={bem.classNames(bem.block, bem.element('sectionPanelSoknader'))}>
+            <Box>
+                <FrontpagePanelWrapper maxColumns={3} title={intlHelper(intl, 'page.dinOversikt.linkPanel.title')}>
+                    <LinkPanel
+                        image={<EttersendIkon />}
+                        title={intlHelper(intl, 'page.dinOversikt.linkPanel.ettersending.title')}
+                        lenke={getLenker().ettersending}
+                        lenkeTekst={intlHelper(intl, 'page.dinOversikt.linkPanel.ettersending.lenkeTekst')}>
+                        <>
+                            <FormattedMessage id="page.dinOversikt.linkPanel.ettersending.lenkeTekst.info" />
+                            <br />
```

Comment:
Prøv å unngå å bruke &lt;br/&gt; tags. Bruk heller CSS for spacing eller &lt;p&gt;